backend/tests/test_auth_api.py

```
@@ -2,160 +1,70 @@
-        "address": "123 Test Street",
-        "phone": "1234567890",
-        "role": "customer",
-    }
-
-    response = client.post("/register", json=user_data)
-
-    print("Response JSON:", response.json())
-    print("Response Status Code:", response.status_code)
-
+from fastapi.testclient import TestClient
+from sqlalchemy.orm import Session
```

Ruff (0.8.2)

`sqlalchemy.orm.Session` imported but unused

Remove unused import: `sqlalchemy.orm.Session`

(F401)